### DISCUSSION TITLE
ADS EAD: duvidas entre faculdades

### DISCUSSION THREAD

**Discussion by clamattos** [frontendbr/forum#1975] [Discussão]
E aí galera, de boa? Seguinte, sou já formado em outra área e migrei para TI recentemente. Hoje trabalho com Cloud (ainda estou aprendendo, no início…) e estou procurando uma boa faculdade EAD para complementar os meus conhecimentos.
Me matriculei na Impacta e apesar de achar o curso bom, senti falta da MATEMÁTICA. Por isso, estou mudando para o Mackenzie (ADS) que tem cálculo e estatística na grade.
Alguém sabe me dizer se a faculdade é boa? Fiz minha primeira graduação lá (administração) e saí bem feliz com o resultado, apesar de não ser a área que sou apaixonado hoje em dia…
E aí, o que me dizem?

OBS. muita gente me falou da FIAP, mas esta um pouco fora do meu orçamento.

**Comment by PedrobyJoao** [comment]
Eaí mano. O que você achou da Mackenzie? Eu estava entra Mackenzie e Impacta também. Mas a Mackenzie parece ter uma grade mais "conservadora"
